src/components/elements/inputs/dropdown.js

```diff
@@ -2,14 +2,10 @@ import React from 'react'
 import Select from 'react-select'
 import Input from './input'
 
-export default class DropdownList extends Input {
+export default class Dropdown extends Input {
```

Comment:
Shouldn't this be called **DropdownList** to be consistent with **BooleanList**?